### ISSUE TITLE
Tunnel enhancement 

### ISSUE THREAD

**Issue by GrantBartlett** [CnCNet/yr-client#29]
<img width="334" alt="screen shot 2015-09-25 at 18 30 39" src="https://cloud.githubusercontent.com/assets/6104940/10107512/94c57460-63b3-11e5-872e-305edb70e9ec.png">

**Comment by Rampastring** [comment]
This would require pinging all the tunnels and storing the result of the ping in memory. Essentially,you'd get the same delay that you have when clicking the "New Game" button when you start the client.

**Comment by mix5003** [comment]
is it possible if pinging all the tunnels in the background thread every 10s or more,  or do once at client start. And when client want to show game information just show only latest time.

**Comment by Rampastring** [comment]
Yes, this could actually be multithreaded so that the UI isn't blocked. I'll try to find time for implementing this at some point.
